chain-storage/src/value.rs

```diff
@@ -72,3 +72,13 @@ impl From<Vec<u8>> for Value {
         Self::owned(value.into_boxed_slice())
     }
 }
+
+impl fmt::Display for Value {
```

Comment:
I'd rather have a custom `Debug` rather than use `Display` for this.
A `Value` could be as large as a block, right?

Reply:
Maybe. The idea was to use it display hex block ids.

Reply:
Perhaps it's better to dump specific type instances after parsing?

Reply:
Yeah probably you're right.